src/styles/card.scss

```diff
@@ -0,0 +1,122 @@
+.card {
+  width: 198px;
+  background-color: white;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  font-family: $roboto-font;
+  color: $text-color;
+  font-weight: 400;
+  transition: all 300ms linear;
```

Comment:
You have a 'transition: all 300ms linear;' property set on the .card class. This means that all changes (including those that may not be intended to be animated) will be transitioned. It's a good practice to specify only those properties that you actually want to animate. For example, if you want to animate the transform and color properties, you should write 'transition: transform 300ms linear, color 300ms linear;'.